documents/policies/external/Competition Requirements.md

```diff
@@ -41,6 +41,7 @@ competition.
       1. The competition website cannot link to web pages that advertise specific products or services of the sponsor (unless it is the sponsor’s homepage).
       2. The competition website cannot advertise any specific products or services of the sponsor, including any endorsements of quality or price.
    3. The competition website may acknowledge the sponsorship contribution.
+10. Competitions with proceeds going to a charity must indicate so on the competition website, preferably in the Information section of the 'General info' tab. The name of the charity must be included. Any such charity needs to be recognized in the country where the competition is held.
```

Comment:
When does this need to be done? Before announcement? Before the competition takes place? What if organizers realize they have an unexpected excess, can they still decide to give this to a charity after the competition has taken place? If yes, what is the remaining purpose of this paragraph?

Reply:
I have currently mentioned this must be done in advance and if done afterwards, WEC must approve it. I will discuss this with WEC about the liability of this.

Reply:
@pokekrom the problem with adding such things _after_ people paid their registration is that they may not want their money going towards that particular charity, and they now have no choice (especially if there is no refund).

Reply:
@c-goodyear Is that a problem though? I would rather argue that once paid, the registration fee is no longer _their money_. It should be sufficient if WEC (or more generally WCA) argrees with a certain charity being a credible choice to send competition excess to.

Reply:
@SAuroux Taking someone's money and then changing what their money is going towards is pretty shifty, especially since I can't think of any charities that are _directly_ in alignment with the WCA's mission/spirit. That's why it's not an issue with funds going towards national/local organizations where the funds will directly assist further WCA competitions.